src/core/server/opensearch/client/client_config.ts

```diff
@@ -75,13 +78,15 @@ export function parseClientOptions(config: OpenSearchClientConfig, scoped: boole
       ...config.customHeaders,
     },
   };
-
-  if (config.pingTimeout != null) {
-    clientOptions.pingTimeout = getDurationAsMs(config.pingTimeout);
+  if (config.memoryCircuitBreaker != null) {
+    clientOptions.memoryCircuitBreaker = config.memoryCircuitBreaker;
   }
   if (config.requestTimeout != null) {
     clientOptions.requestTimeout = getDurationAsMs(config.requestTimeout);
   }
+  if (config.pingTimeout != null) {
```

Comment:
nit: not the biggest of deal but my recommendation would have been to keep ping timeout where it was and just move the memoryCircuitBreaker config logic on top. to reduce the delta change  within this PR.

Reply:
Done: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1463